pkg/api/types.go

```diff
@@ -66,6 +69,19 @@ type ReleaseBuildConfiguration struct {
 	Resources ResourceConfiguration
 }
 
+// ReleaseToolsConfig holds configuration for tools used in the infrastructure
+type ReleaseToolsConfig struct {
+	// ClonerefsSourceImage configures from which image we extract the clonerefs
+	// tool for use in our clone steps
+	ClonerefsSourceImage ClonerefsSourceImage `json:"clonerefs_source_image"`
```

Comment:
Are we really, really, really going to want to have multiple of these?  Why wouldn't we just have a release tools image.